app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,42 +33,103 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left w-1/6 text-grey-tableTitle text-tableTitle">
+              Title
+            </th>
+            <th className="text-left w-1/5 text-grey-tableTitle text-tableTitle">
+              Description
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Tags
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Created Date
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Due Date
+            </th>
+            <th className="text-center w:auto text-grey-tableTitle text-tableTitle">
+              Contact
+            </th>
+            <th className="text-center w-1/12 text-grey-tableTitle text-tableTitle"></th>
```

Comment:
I understand that you have used custom styles to match the design, however, please follow the default styles which come with neetoUI `table.scss`. No need to override them here.

Reply:
Resolved in #9